examples/bri-3/prisma/migrations/20240111112712_add_public_key_to_bpi_subject/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Changed the type of `publicKey` on the `BpiSubject` table. No cast exists, the column would be dropped and recreated, which cannot be done if there is data, since the column is required.
+
+*/
+-- AlterTable
+ALTER TABLE "BpiSubject" DROP COLUMN "publicKey",
+ADD COLUMN     "publicKey" JSONB NOT NULL;
```

Comment:
why storing json { pk1, pk2 } instead of having pk1 and pk2 as separate fields? storing json seems like it can just introduce not needed overhead to always parse and unparse

Reply:
I will remove the JSON and instead create another table for storing public keys.